bankaccounts.rb

```diff
@@ -88,5 +84,4 @@ def self.find(id)
     end
```

Comment:
In the `all` method above, I'd use a more explicit variable name rather than `o` in your `map` method call